core/arch/arm/plat-imx/drivers/tzc380.c

```diff
@@ -25,7 +26,23 @@
 register_phys_mem(MEM_AREA_IO_SEC, TZASC2_BASE, TZASC_SIZE);
 #endif
 
-register_phys_mem(MEM_AREA_IO_SEC, TZASC_BASE, TZASC_SIZE);
```

Comment:
I'm confused. Doesn't this temporarily break things until the next commit? Why does this have to be removed and added back later?